rails-commenting.rb

```diff
@@ -48,15 +48,15 @@ def update
   def destroy
     @post = BlogPost.find(params[:id])
     if @post.destroy
-      # ---8)
+      # ---8)This is saying if our blog post is deleted, then its going to redirect us to blog posts path we've created.
       redirect_to blog_posts_path
     end
   end
 
-  # ---9)
+  # ---9)This is where we designate our strong parameters, which dictate what information can be passed into the database and checks for validity of each entry and its corresponding values that we haev set in our model. 
```

Comment:
The private keyword makes anything below it only available in the scope of this controller